prem_utils/models.json

```diff
@@ -189,7 +460,35 @@
                     "input_cost_per_token": 0.00001,
                     "output_cost_per_token": 0.00003,
                     "alias": "gpt-4-turbo",
-                    "group": "gpt"
+                    "group": "gpt",
+                    "parameters":
+                    {
+                        "temperature": {
+                            "default": 1.0,
+                            "min": 0.0,
+                            "max": 2.0
+                        },
+                        "max_tokens": {
+                            "default": 4096,
+                            "min": 0,
+                            "max": 4096
```

Comment:
why max_tokens less than context window?

Reply:
I tested gpt-4-turbo-preview with prem-utils connector and also with openAI chat API. And I figured out that 4096 is the upper limit. Although, openAI [docs ](https://platform.openai.com/docs/api-reference/chat/create) are not super clear on this